docs/design/queue/queue-priority-design.md

```diff
@@ -0,0 +1,418 @@
+Proportion Weight = 1
+```
+|        | Priority | DRF  Share | Proportion Share | Priority Score | DRF Score | Proportion Score | Total Score |
+|--------|----------|------------|------------------|----------------|-----------|------------------|-------------|
+| QueueA | 40       | 0.3        | 1.2              | 0.4            | 0.7       | -0.2             | 0.9         |
+| QueueB | 80       | 0.4        | 0.5              | 0.8            | 0.6       | 0.5              | 1.9         |
+| QueueC | 0        | 0.5        | 0.3              | 0              | 0.5       | 0.7              | 1.2         |
+
+QueueScoreOrder： **Queue B > Queue C > Queue A**
+
+### Impact priority of job
+
+Due to the Priority of queue, therefore, **if priority not setting in job, the job priority will inherit the priority of queue**.
+
+*Through the idea of QueueScore, here is a reflection：*
+
+At present, all of enqueue、 allocate or reclaim actions are sort the jobs in the queue after the queue is sorted.
+If the following situation occur.
+
+There are two queue:
+
+Queue A(priority 10) has A1 Job(default priority 10), A2 Job(priority 12).
+
+Queue B(priority 8) has B1 Job(default priority 8), B2 Job(priority 13).
+
+Current logic of the three actions are as follows:
+* A Queue
+    * A2 Job
+    * A1 Job
+* B Queue
+    * B2 Job
+    * B1 Job
+
+Try to think another way, if we think job is the base unit in scheduler, priority of job maybe more important than priority of queue.
+
+New logic of the three actions are as follows:
+* B2 Job
+* A2 Job
+* A1 Job
+* B1 Job
+
+And the QueueOrderFn in drf or proportion will be a factor affecting JobOrder.
+Of course, the perspectives of these two methods are different, but there is no problem in essence.
+
+## Design
+### API
+```
+apiVersion: scheduling.volcano.sh/v1beta1
+kind: Queue
+metadata:
+  name: q1
+spec:
+  reclaimable: true
+  weight: 1
+  priorityClassName: system-cluster-critical
+```
+
+The priority configuration of the queue follows the native PriorityClass of Kubernetes.
+
+### Scheduler configuration
+```
+actions: "enqueue, allocate, backfill"
+configurations:
+- name: enqueue:
+  arguments:
+    QueueScoreOrderEnable: true
+    QueueScoreOrder.priority.weight: 1 # key: QueueScoreOrder.pluginName.weight
+    QueueScoreOrder.drf.weight: 1
+    QueueScoreOrder.proportion.weight: 1
+- name: allocate:
+  arguments:
+    QueueScoreOrderEnable: true
+    QueueScoreOrder.priority.weight: 1
+    QueueScoreOrder.drf.weight: 1
+    QueueScoreOrder.proportion.weight: 1
+- name: reclaim:
+  arguments:
+    QueueScoreOrderEnable: true
+    QueueScoreOrder.priority.weight: 1
+    QueueScoreOrder.drf.weight: 1
+    QueueScoreOrder.proportion.weight: 1
+tiers:
+- plugins:
+  - name: priority
+  - name: gang
+    enablePreemptable: false
+  - name: conformance
+- plugins:
+  - name: overcommit
+  - name: drf
+  - name: predicates
+  - name: proportion
+  - name: nodeorder
+  - name: binpack
+```
+
+
+## Implementation
```

Comment:
It is enough to see the scenarios, motivtion, feature desgin and API design in the documention. Implementation is not necessary and it's better to include that in the PR. Thanks.